gradle.properties

```diff
@@ -1,2 +1,3 @@
 version = 2.3-SNAPSHOT
 githubUser = jenkinsci
+org.gradle.jvmargs=-Xmx1g -XX:MaxMetaspaceSize=512m
```

Comment:
Keep in mind that this is also applied to any CI agents where this project is running on. Does it mean that with Gradle 8 you get any kind of OOM errors?

Reply:
Yes, exactly, with the standard memory limits it was not working on jenkins.